### PR TITLE
Basically a few fixes

### PR DESCRIPTION
**Pull request by Villiamfj**
Upped the buffer to u32 for package sizes for Client. 
Added Range map to measurement
Renamed config and service to server

### CODE REVIEW

File: crates/remote-client/src/main.rs [hunk 1]
```diff
@@ -52,7 +53,8 @@ async fn main() -> Result<(), Box<dyn Error>> {
 
         let remote_client_packets: Vec<RemoteClientPacket> =
             bincode::deserialize(&client_buffer).unwrap();
-        println!("Remote client packet: {:?}", remote_client_packets);
+        //println!("Remote client packet: {:?}", remote_client_packets);
+        println!("writting to csv");
```
Comment (Jakobzs) [inline crates/remote-client/src/main.rs:57]:
```suggestion
        println!("Writing to csv");
```

File: crates/server/src/main.rs [hunk 2]
```diff
@@ -22,27 +22,25 @@ use std::{fs, sync::Arc};
...
     // waiting for Sampler to begin
     sleep(std::time::Duration::from_secs(1));
 
     let listen = ListenerImplem {
         ip: config.thor.server_ip.clone(),
-        remote_packet_queue_cycle: config.thor.remote_packet_queue_cycle,
+        remote_packet_queue_cycle: config.thor.remote_packet_queue_cycle_millis,
     };
-    let _ = listen.start_listening(start, build, measure);
+    let _ = listen.start_listening(start, build, &mut measure);
```
Comment (Jakobzs) [inline crates/server/src/main.rs:45]: Necessary to `let _ = ...`?

File: crates/server/src/measurement.rs [hunk 3]
```diff
@@ -18,50 +18,69 @@ use tokio::{io::AsyncReadExt, net::TcpListener};
...
 
 impl RaplSampler {
-    pub fn start_sampling(&self, sampling_interval: u64) -> Result<()> {
+    pub fn new(max_sample_age: u128, sampling_interval: u64) -> RaplSampler {
+        let result = RaplSampler {
+            max_sample_age,
+            range_map: RangeMap::new(),
+            sampling_interval,
+        };
+        let _ = result.start_sampling(sampling_interval);
```
Comment (Jakobzs) [inline crates/server/src/measurement.rs:64]: Here too with `let _ = ...`

**Review by Jakobzs — CHANGES_REQUESTED** [review]

File: crates/server/src/main.rs (hunk 2 above)
Comment (Villiamfj) [reply crates/server/src/main.rs]: changed to unwrap

File: crates/server/src/measurement.rs (hunk 3 above)
Comment (Villiamfj) [reply crates/server/src/measurement.rs]: changed to unwrap

**Pull request merged**